strategy/plan2021.md

```diff
@@ -0,0 +1,134 @@
+      number of staff ideally needed to succeed in 2 weeks, and
+      stakeholders
+  - Team Members vote on "project" ideas for the current cycle
+    - Select highest voted projects based on staffing available and
+      staffing needed (e.g., if 3 projects are pitched with staffing
+      needs of 2, 3, and 2, and the team consists of 5 people, then
+      only the 2 projects with the highest votes are chosen)
+    - Team Members choose which project they want to work on for
+      this cycle (equitable selection method, not first-come-first-serve)
+    - Team Member(s) who pitched the project are selected as
+      "Project Leads"
+    - Projects that are not selected for this cycle retain their
+      votes for the next cycle (i.e., they don't start with zero
+      votes at the next Pitch Meeting)
+- *Project Planning* (Remaining Week)
+  - Project Leads prepare a "deep dive" session on the necessary
+    code, and give pointers to Project Team Members about what code
+    they should familiarize themselves with before the "deep dive"
+    session (this is hard!!!)
+  - Project Leads prepare "Project Specification" documents
+- Monday, Week 2: *Project Presentations Meeting*
+  - Team Leads go over project specification document, goals and
+    point Project Team to code they should get familiar with
+- *Team Pre-Learning* (Next 2 Days)
+  - Project Team explores and tries to come up to speed with
+    material given to them in the Presentation Meeting
+- Wednesday, Week 2: *Project Deep Dive Meetings*
+  - Project Leads give "deep dive" training sessions to the rest of the
+    Project Team
+  - Project Leads indicate goals and discuss project plans (i.e.,
+    contents of the Project Specificaion document)
+  - Project Team Members volunteer for tasks
+- *Project Work Cycle* (2 Weeks)
+  - Project Leads and Team collaborate on project work, set up necessary
+    hack sessions with each other, pair programming, etc.
+  - Work proceeds until the work cycle is complete (early completed projects
+    can add additional tasks and ammend the Project Spec, projects that
+    don't complete just indicate their progress and plans for future)
+- Wednesday, Week 4: *Project Summary Meeting*
+  - Project Team presents results, conclusions, lessons learned, remaining
+    problems to solve and next steps
+  - Project Completion Meeting immediately followed by *Project Pitch Meeting*
+    on the following Monday
+
+## Activities: *What we do*
+
+Many ideas for projects were given in the *Activities* section of the
+Start/Stop/Continue exercise, and many continuing Activities were listed
+as high priority.  No Activities were lists in the Stop category, indicating
+*to me* that what we feel we need to do most is be more productive.  Hence,
+for future reference, I include all of the mentioned activities in prioritized
+order by vote count.
+
+- **[OLD]** More collaborations with the GeoCAT team like the GeoCAT Visualization Gallery Hackathon [9]
```

Comment:
I am glad and feeling responsible to see such a high number of votes from the Xdev team to increase collaboration with GeoCAT. The GeoCAT team is sharing the same motivation. Methods of how to achieve these could be discussed in the upcoming project cycles of course; however, I just want to write down a few channels we can make use of currently (which @pilotchute has likely have shared in the weekly meeting):

- GeoCAT has been holding weekly hack days on every Friday (between 10am - 3pm every other Friday, between 1 pm - 3 pm rest of the every other Fridays), where we discuss/share, e.g.:
    - Newly learned technologies
    - Technical stuff that any team member is recently struggling 
    - Any issue that needs to be discussed as a team,
    - Plan of a new development approach
    - Etc.
In these events, many topics, which would be interest of both teams (such as Xarray, Dask, code coverage, code quality, CI, etc.), are being discussed and worked hands-on when needed. I am happy to have these weekly events all together with Xdev. If a weekly frequency is more than enough, we can dedicate at least one of them monthly to our collaboration.

Let us further discuss this and other possibilities.

Reply:
While I personally do not work on Fridays, I think many Xdev members would love to be invited to those hack days.  It's a great way of concentrating our collaborative efforts!